c2corg_api/views/forum.py

```diff
@@ -82,7 +82,8 @@ def collection_post(self):
         title = "{}_{}".format(locale.document_id, locale.lang)
         content = '<a href="{}">{}</a>'.format(
                 self.request.referer,
-                locale.title)
+                locale.title or "{}/{}".format(locale.document_id,
```

Comment:
OK for using ``document_id/lang`` as fallback even though it might not be very clear for users to have a link such as ``123456/fr`` :p 

Would it be possible to add the document type as well? For instance ``images/123456/fr`` ?